docs/sdk/js/cli/cli.md

```diff
@@ -0,0 +1,191 @@
+---
+title: JS SDK CLI
+---
+
+The SDK [Command Line Interface](https://github.com/near/near-sdk-js/blob/develop/packages/near-sdk-js/src/cli/cli.ts) (CLI) is a tool that enables to act on different parts of the build process as well as generate validations and an [ABI](https://github.com/near/abi). Among other things, the SDK CLI enables you to:
+
+- Control the different parts of the build process
+- Validate your contract and TypeScript code
+- Create an ABI JSON file
+
+---
+
+## Overview {#overview}
+
+_Click on a command for more information and examples._
+
+**Commands**
+
+| Command                                   | Description                                                                                                                    |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| [`near-sdk-js build`](#build)               | stores a full access key locally using [NEAR Wallet](https://wallet.testnet.near.org/) |
+| [`near-sdk-js validateContract`](#validate-contract)                 | displays all access keys and their details for a given account |
+| [`near-sdk-js checkTypescript`](#check-ts) | generates a local key pair **or** shows public key & [implicit account](/concepts/basics/accounts/account-id#implicit-accounts) |
+| [`near-sdk-js createJsFileWithRollup`](#create-js-file)           | adds a new access key to an account |
+| [`near-sdk-js transpileJsAndBuildWasm`](#transpile-js-to-wasm)     | deletes an access key from an account |
```

Comment:
Maybe I'm mistaken, but the descriptions under the "Description" column seem out of place with the actual commands on the left. (Example: "transpile js" >> "deletes an access key from an account")

Reply:
Ah well spotted! Let me fix that.